src/main/java/de/hysky/skyblocker/skyblock/itemlist/recipebook/SkyblockCraftingRecipeResults.java

```diff
@@ -194,7 +195,8 @@ public void updateSearchResults(String query, FilterOption filterOption, boolean
 			for (ItemStack stack : ItemRepository.getItems()) {
 				String name = stack.getName().getString().toLowerCase(Locale.ENGLISH);
 				if (!filterOption.test(name)) continue;
-				List<Text> lore = ItemUtils.getLore(stack);
+
+				List<Text> lore = TooltipManager.addToTooltip(null, stack, new ArrayList<>(ItemUtils.getLore(stack)));
```

Comment:
causes a crash after opening the recipe book.
might be fixable with the following change:
```suggestion
				List<Text> lore = Screen.getTooltipFromItem(MinecraftClient.getInstance(), stack);
```